networkit/cpp/centrality/test/DynBetweennessGTest.cpp

```diff
@@ -16,6 +16,9 @@
 #include <networkit/graph/Sampling.hpp>
 #include <networkit/generators/DorogovtsevMendesGenerator.hpp>
 #include <networkit/generators/ErdosRenyiGenerator.hpp>
+#include <networkit/centrality/DynBetweenness.hpp>
+#include <networkit/auxiliary/Random.hpp>
+#include <random>
```

Comment:
```suggestion
```
`<random>` is not needed, you can just use `<networkit/auxiliary/Random.hpp>` to extract random nodes.